lib/common/options.c

```diff
@@ -578,14 +578,14 @@ pcmk__print_option_metadata(const char *name, const char *version,
                 "      <longdesc lang=\"en\">%s%s%s</longdesc>\n"
                 "    </parameter>\n",
                 option_list[lpc].name,
-                option_list[lpc].description_short,
+                _(option_list[lpc].description_short),
                 option_list[lpc].type,
                 option_list[lpc].default_value,
                 option_list[lpc].description_long?
-                    option_list[lpc].description_long :
-                    option_list[lpc].description_short,
-                (option_list[lpc].values? "  Allowed values: " : ""),
-                (option_list[lpc].values? option_list[lpc].values : ""));
+                    _(option_list[lpc].description_long) :
+                    _(option_list[lpc].description_short),
+                (option_list[lpc].values? _("  Allowed values: ") : ""),
+                (option_list[lpc].values? _(option_list[lpc].values) : ""));
```

Comment:
I did some more research. Normally it's best to use \_() around the literal string, rather than a variable that uses it. However that does not work with static strings since they are defined at compile-time, which I assume is why you went with this approach above.

Looking at the documentation, we still need to mark the literal strings, but with a no-op macro. gettext.h already defines gettext_noop(), and Makevars already looks for N\_(), so all we need to do is define N\_() to gettext_noop() (like \_() is defined to gettext()) and then mark all the static strings to be translated with N\_().